### PR TITLE
Hmpid dcsccdb improvments

### PR DESCRIPTION
**Pull request by fapfap69**

### CODE REVIEW

File: Detectors/HMPID/workflow/src/PedestalsCalculationSpec.cxx [hunk 1]
```diff
@@ -229,11 +241,8 @@ void PedestalsCalculationTask::recordPedInDcsCcdb()
     o2::dcs::addConfigItem(pedestalsConfig, "Equipment" + std::to_string(e), (const char*)outBuffer);
   }
 
-  struct timeval tp;
-  gettimeofday(&tp, nullptr);
-  uint64_t ms = tp.tv_sec * 1000 + tp.tv_usec / 1000;
-  uint64_t minTimeStamp = ms;
-  uint64_t maxTimeStamp = ms + 1;
+  long minTimeStamp = o2::ccdb::getCurrentTimestamp();
+  long maxTimeStamp = o2::ccdb::getFutureTimestamp(60 * 60 * mDcsCcdbAliveHours * 1);
```
Comment (shahor02) [inline Detectors/HMPID/workflow/src/PedestalsCalculationSpec.cxx]: This should be an absolute end-validity time in ms, i.e. `maxTimeStamp = minTimeStamp + 3600L * DcsCcdbAliveHours * 1000;`

File: Detectors/HMPID/workflow/src/PedestalsCalculationSpec.cxx [hunk 2]
```diff
@@ -289,11 +298,9 @@ void PedestalsCalculationTask::recordPedInCcdb()
...
     }
   }
-  struct timeval tp;
-  gettimeofday(&tp, nullptr);
-  uint64_t ms = tp.tv_sec * 1000 + tp.tv_usec / 1000;
-  uint64_t minTimeStamp = ms;
-  uint64_t maxTimeStamp = ms + 1;
+
+  long minTimeStamp = o2::ccdb::getCurrentTimestamp();
+  long maxTimeStamp = o2::ccdb::getFutureTimestamp(60 * 60 * 24 * (5 * 365)); // 5 years
```
Comment (shahor02) [inline Detectors/HMPID/workflow/src/PedestalsCalculationSpec.cxx]: same here

**Review by shahor02 — CHANGES_REQUESTED** [review]
Hi @fapfap69 

Please see comments below, also, the PR fails due to the clang-format.

Cheers,
 Ruben

**Comment by fapfap69** [comment]
@shahor02 I did the clang-format fix, now seems that all checks passed

**Review by shahor02 — APPROVED** [review]

**Pull request merged**